### PR TITLE
Use select list for metadata languages

### PR DESCRIPTION
**Pull request by Beacze**

### CODE REVIEW

File: Kitodo/src/main/java/org/kitodo/production/forms/UserForm.java [hunk 1]
```diff
@@ -386,6 +388,18 @@ public void load(int id) {
...
     }
 
+    /**
+     * Get map of supported metadata languages.
+     * 
+     * @return map of supported metadata languages
+     */
+    public Map<String, String> getMetadataLanguages() {
+        Map<String, String> metadataLanguages = new HashMap<>();
+        metadataLanguages.put("English", "en");
```
Comment (Kathrin-Huber) [inline Kitodo/src/main/java/org/kitodo/production/forms/UserForm.java]: This seems to be pretty hardcoded, is there any way to make it configurable?

Comment (Beacze) [reply Kitodo/src/main/java/org/kitodo/production/forms/UserForm.java]: I don't think so. I don't know if there is some list of available languages which could be used here.

Comment (henning-gerhardt) [reply Kitodo/src/main/java/org/kitodo/production/forms/UserForm.java]: @Kathrin-Huber , @matthias-ronge, @Erikmitk :
Maybe it is possible to move the translation of the meta data and structure fields out of the ruleset files in translation files. The internal field names are used then as translation keys and translation is easily maintained out of ruleset files. Adding new language translations require then only an added translation bundle file and no change to the ruleset file.

Comment (Kathrin-Huber) [reply Kitodo/src/main/java/org/kitodo/production/forms/UserForm.java]: > 
> 
> I don't think so. I don't know if there is some list of available languages which could be used here.

Can you add it to the config file? "metadataLanguages" or something

Comment (Beacze) [reply Kitodo/src/main/java/org/kitodo/production/forms/UserForm.java]: Added.

**Pull request merged**